src/components/Resource/ResourceDetailsUpdate.tsx

```diff
@@ -192,123 +174,158 @@ export const ResourceDetailsUpdate = (props: resourceProps) => {
+          <Form
+            className=""
+            defaults={state.form}
+            onSubmit={handleSubmit}
+            onCancel={() => goBack()}
+            noPadding
+            hideRestoreDraft
+          >
+            {(field) => (
+              <div className="grid grid-cols-1 gap-4 md:grid-cols-2">
+                <div className="md:col-span-1">
+                  <SelectFormField
+                    label="Status"
+                    name="status"
+                    value={field("status").value}
+                    options={resourceStatusOptions}
+                    onChange={(e: any) => field("status").onChange(e)}
+                    optionLabel={(option) => option}
                   />
-                )}
-              </div>
-            </div>
-            <div>
-              <FieldLabel>Name of resource approving facility</FieldLabel>
-              <FacilitySelect
-                multiple={false}
-                name="approving_facility"
-                facilityType={1500}
-                selected={state.form.approving_facility_object}
-                setSelected={(obj) =>
-                  setFacility(obj, "approving_facility_object")
-                }
-                errors={state.errors.approving_facility}
-              />
-            </div>
-
-            <div>
-              <FieldLabel>
-                What facility would you like to assign the request to
-              </FieldLabel>
-              <FacilitySelect
-                multiple={false}
-                name="assigned_facility"
-                facilityType={1510}
-                selected={state.form.assigned_facility_object}
-                setSelected={(obj) =>
-                  setFacility(obj, "assigned_facility_object")
-                }
-                errors={state.errors.assigned_facility}
-              />
-            </div>
-            <div>
-              <TextFormField
-                label="Required Quantity"
-                name="requested_quantity"
-                type="number"
-                value={state.form.requested_quantity}
-                onChange={handleChange}
-              />
-            </div>
-            <div>
-              <TextFormField
-                name="assigned_quantity"
-                type="number"
-                label="Approved Quantity"
-                value={state.form.assigned_quantity}
-                onChange={handleChange}
-                disabled={state.form.status !== "PENDING"}
-              />
-            </div>
+                </div>
+                <div className="md:col-span-1">
+                  <div className="">
+                    {assignedUserLoading ? (
+                      <CircularProgress />
+                    ) : (
+                      <UserAutocomplete
+                        label="Assigned To"
+                        value={field("assigned_to_object").value}
+                        onChange={(e: any) => {
```

Comment:
remove `any` type here too and all other places that you've added/modified

Reply:
sure